manifests/u/Ultra/Ultra/1.0.0.4/Ultra.Ultra.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Ultra.Ultra
+PackageVersion: 1.0.0.4
+PackageLocale: en-US
+Publisher: Ultra

```

```suggestion
Publisher: The Ultra Authors
```
This must match Control Panel